packages/trader/src/App/Components/Elements/VideoPlayer/playback-rate-control.tsx

```diff
@@ -48,4 +48,4 @@ const PlaybackRateControl = ({ onPlaybackRateChange, is_mobile, playback_rate }:
     );
 };
 
-export default PlaybackRateControl;
+export default React.memo(PlaybackRateControl);
```

Comment:
re-renders:
- before: 4+ times per second during 1 video playing cycle;
- after: on the 1st video render and only when another playback_rate is set